.babelrc

```diff
@@ -6,5 +6,9 @@
     ],
     "plugins": [
```

Comment:
Added the translation plugin using npm i --save-dev babel-plugin-i18next-extract